code/game/objects/items/devices/broadcast_camera.dm

```diff
@@ -57,10 +57,17 @@
 	. = ..()
 	. += span_notice("Broadcast name is <b>[broadcast_name]</b>")
 
-/// When wielding the camera
-/obj/item/broadcast_camera/proc/on_wield()
+/obj/item/broadcast_camera/on_enter_storage(datum/storage/master_storage)
+	. = ..()
+	if(active)
+		on_deactivating()
```

Comment:
This does work, but this still doesn't prevent being able to drop the camera after turning it on, this is still an issue as the _actual camera_ still spawns _inside_ of you.

So people will still be able to turn the the camera on, drop it somewhere, and be a walking camera.

Reply:
Drop check added. It's a little sad that the camera code doesn't work well because you can't place the camera in front of you and film something like a review on a webcam

Reply:
I would suggest making it work when dropped only if you're pulling it.

But really I would just make a separate PR to add a tripod to put a camera on or smt.